src/SingleArticle.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react"
+import {getArticle} from './api'
+
+function SingleArticle({currentArticle}){
```

Comment:
passing down the article_id as a prop is fine here, but if you'd like to make the url dynamic then you can use useParam() to get the article_id here